middlewares/errorHandler.js

```diff
@@ -0,0 +1,7 @@
+exports.notFoundErrorHandler = (req, res, next) => {
+  const error = {
+    status: 404,
+    message: "API endpoint does not exists",
+  };
+  next(error);
```

Comment:
Can you please directly send response without actually sending it to the next handler...